gui/wxpython/dbmgr/base.py

```diff
@@ -965,7 +965,7 @@ def OnLayerPageChanged(self, event):
             pass
 
         if idCol:
-            winCol = self.FindWindowById(idCol)
+            self.FindWindowById(idCol)
```

Comment:
There are no side effects to the wxPython's [FindWindowById function](https://docs.wxpython.org/wx.functions.html#wx.FindWindowById), so there is no reason to call it when the return value is unused.